payments/templates/payments/sap/sales_order.xml

```diff
@@ -0,0 +1,47 @@
+{% load l10n %}
+<?xml version="1.0" encoding="UTF-8"?>
+<SalesOrder>
+{% for order in orders %}
+    <header>
+        <Customer>
+        <SapID></SapID>
+        <Company>
+        <BusinessID>{{ order.business_id }}</BusinessID>
+        <CompanyName>{{ order.company_name }}</CompanyName>
+        </Company>
+        <address>
+            <Street>{{ order.address.street }}</Street>
+            <Town>{{ order.address.town }}</Town>
+            <PostCode>{{ order.address.postcode }}</PostCode>
+        </address>
+        </Customer>
+        <BillingDate>{{ order.billing_date }}</BillingDate>
+        <ServiceRenderDate>{% now "Y-d-m" %}</ServiceRenderDate>
+        <BillNumber>20186245</BillNumber>
+        <YourReference>{{ order.reference }}</YourReference>
+        <SalesOrganisation>1111</SalesOrganisation>
+        <DistributionChannel>00</DistributionChannel>
+        <Division>00</Division>
+        <SalesOrderType>ZXXX</SalesOrderType>
+        <InterfaceID>XXX</InterfaceID>
+        <Text>
+        <TextRow>{{ billing_period }}</TextRow>
+        </Text>
+        <Items>
+        {% for item in order.items %}
+            <Item>
+            <Description>{{ item.description }}</Description>
+            <ProfitCenter>{{ item.cost_center_code }}</ProfitCenter>
+            <InternalOrder>000000027070</InternalOrder>
+            <Material>{{ item.material }}</Material>
+            <PriceCondition>ZMYH</PriceCondition>
+            <Currency>{{ item.currency }}</Currency>
+            <Quantity>{{ item.quantity }}</Quantity>
+            <UnitPrice>{{ item.unit_price|unlocalize }}</UnitPrice>
```

Comment:
Decimal format should be in international standard e.g. "12.50" not "12,50" for "twelve euro 50 cent".